core/templates/domain/statistics/PlaythroughObjectFactorySpec.ts

```diff
@@ -58,6 +58,31 @@ describe('Playthrough Object Factory', () => {
     expect(playthroughObject.actions).toEqual(actions);
   });
 
+  it('should create same objects from backend dict and direct values.', () => {
+    var playthroughDictObject = pof.createFromBackendDict(
+      {
+        exp_id: 'expId1',
+        exp_version: 1,
+        issue_type: 'CyclicStateTransitions',
+        issue_customization_args: {
+          state_names: {
+            value: ['state1', 'state2']
+          }
+        },
+        actions: []
+      }
+    );
```

Comment:
Please collapse brackets when they don't need line breaks, like so:

```suggestion
    var playthroughDictObject = pof.createFromBackendDict({
      exp_id: 'expId1',
      exp_version: 1,
      issue_type: 'CyclicStateTransitions',
      issue_customization_args: {
        state_names: {
          value: ['state1', 'state2']
        }
      },
      actions: []
    });
```

Reply:
done